packages/web-components/fast-components/src/switch/switch.styles.ts

```diff
@@ -46,7 +41,6 @@ export const switchStyles: (
     :host([hidden]) {
         display: none;
     }
-
```

Comment:
Line 48 below adds margin to the host element - it looks like this has been here, but this should be removed. It should likely be removed in a separate PR from the existing control though. This is problematic as margin relates to space between things which should be applied contextually.